src/scene/materials/standard-node-material.js

```diff
@@ -0,0 +1,189 @@
+import { standard } from '../../graphics/program-lib/programs/standard.js';
+
+import {
+    SHADER_FORWARDHDR, SHADER_PICK
+} from '../constants.js';
+
+import { StandardMaterial } from './standard-material.js';
+
+/**
+ * @private
+ * @class
+ * @name pc.StandardNodeMaterial
+ * @augments pc.StandardMaterial
+ * @classdesc StandardNodeMaterial is sub class of the StandardMaterial, and adds shader graph
+ * interop functionality.
+ * @param {pc.StandardMaterial} mat - Optional material which is cloned.
+ * @param {any} chunk - Optional shader graph node chunk to be used.
```

Comment:
does it take "any" parameter?

Reply:
It should be ShaderGraphNode - I'll update it

Reply:
updated